repo_commits.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2014 The Gogs Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package gogs
+
+import (
+	"fmt"
+	"time"
+)
+
+type Signature struct {
```

Comment:
It is same as https://github.com/gogits/go-gogs-client/blob/master/repo_hooks.go#L75